### ISSUE TITLE
Spinners not included in 7.0.2 release

### ISSUE THREAD

**Issue by hassan-yonomi** [reactstrap/reactstrap#1361]
- components: `spinner`
- reactstrap version `7.0.2`
- import method `umd/csj/es` 
- react version `16.7.0`
- bootstrap version `4.2.1`

### What is happening?

The latest release (7.0.2) installing it through npm dosent include spinners. It seems to be the npm package and the latest in git are out of sync ?

### What should be happening?
 
Installing the latest through NPM should include spinners

**Comment by TheSharpieOne** [comment]
They are on master, master is the latest. Check the [releases on github](https://github.com/reactstrap/reactstrap/releases) for what is released.

**Comment by IkechukwuAKalu** [comment]
I'm having the same issue. When I check the source code on Github, I see the `Spinner` file there. But when I check it in the `node_modules` directory, I don't see it there. Any help on this? Thanks.

**Comment by TheSharpieOne** [comment]
**tl;dr**: It's not yet released, it will be in the next release (either `7.1.0` or `8.0.0`)

**Comment by TommyBakkenMydland** [comment]
@TheSharpieOne 
Do you know when the next release will be that include the spinner component?

**Comment by noise-machines** [comment]
Thanks for your hard work on reactstrap, @TheSharpieOne! It's been a pleasure to use so far. I really appreciate how well designed the API is and how good everything looks out of the box. I especially like how it uses child components instead of passing everything through props. Makes the JSX much more natural and readable.

Just wanted to chime in that I got bit by this as well. It's especially confusing since spinners are [listed in the official documentation](https://reactstrap.github.io/components/spinners/). As a quick and dirty solution, maybe put a notice on the docs that they represent master, not the latest release?

Cheers,
Thomas

**Comment by AbdullAlSaleh** [comment]
does anyone have these spinner working! if yes, please let me know how!

**Comment by IkechukwuAKalu** [comment]
Yeah @AbdullAlSaleh, the Spinners are working fine as of version `7.1.0`. You can try it out
